src/rhoknp/units/morpheme.py

```diff
@@ -309,7 +316,8 @@ def _from_jumanpp_line(cls, jumanpp_line: str, homograph: bool = False) -> "Morp
         """
         assert "\n" not in jumanpp_line.strip("\n")
         if (match := cls.JUMANPP_PAT.match(jumanpp_line)) is None:
```

Comment:
Why not `match := cls.JUMANPP_PAT.match(jumanpp_line) or cls.JUMANPP_PAT_REPEATED.match(jumanpp_line)`?